nuklear-sys/build.rs

```diff
@@ -24,12 +33,36 @@ fn main() {
         .trust_clang_mangling(false)
         .derive_default(true)
         // Generate bindings for the wrapper
-        .header("wrapper.h")
+        .header("nuklear_include.h")
         // Tell cargo to invalidate the built crate whenever any of the included header files changed.
         .parse_callbacks(Box::new(bindgen::CargoCallbacks))
         .rustfmt_bindings(true)
+        .parse_callbacks(Box::new(MacroCallback { macros: Arc::new(RwLock::new(HashSet::new())) }))
         .generate()
         .expect("Unable to generate bindings");
 
+    let dest_path = out_path.join("bindings.rs");
     bindings.write_to_file(dest_path).expect("Couldn't write bindings!");
 }
+
+#[derive(Debug)]
+struct MacroCallback {
+    macros: Arc<RwLock<HashSet<String>>>,
+}
+
+impl ParseCallbacks for MacroCallback {
+    fn will_parse_macro(&self, name: &str) -> MacroParsingBehavior {
+        self.macros.write().unwrap().insert(name.into());
+
+        if name == "FP_INFINITE"
```

Comment:
This is needed because there are `#define`s and `enum` with the same names in the Nuklear code, but both do not need to be exported.